tests/Unit/Endorsement/SpecialEndorsementRequestServiceTest.php

```diff
@@ -86,15 +86,21 @@ public function itCanActionRequestsForSpecialEndorsements()
         $this->mock(User::class, function ($mock) {
             $mock->shouldReceive('find')
                 ->andReturn(
-                    User::initModelWithData([
+                    new User([
+                        'id' => 1300005,
+                        'name_first' => 'Callum',
+                    ])
+                );
+            $mock->shouldReceive('initModelWithData')
```

Comment:
Is this still current? I see you've removed the majority of the references to this method elsewhere.

Reply:
I was a bit inexperienced with Laravel a few days ago and one of the tests was failing with the initModelWithData mock, so I thought "ah let's just get rid of it, it'll work fine". I'll probably add back the references